packages/schema/build.config.ts

```diff
@@ -27,6 +27,6 @@ export default defineBuildConfig({
     'vite',
     // Implicit
     '@vue/compiler-core',
-    '@vue/shared ',
+    '@vue/shared '
```

Comment:
This isn't caused by this PR - but surely:
```suggestion
    '@vue/shared'
```